HelpAssistHer/help-assist-her
# verification portal heading removed

## Description
- The top heading is removed and Login button is still available on the right side
- The gray background is removed and the background is completely white like designs
- VERIFICATION PORTAL heading is removed

## Test Plan
- yarn webpack 
- visit localhost/4000
- verify all changes
